Source/ocdm/adapter/open_cdm_adapter.h

```diff
@@ -88,6 +88,45 @@ extern "C" {
     
     EXTERNAL OpenCDMError opencdm_gstreamer_session_decrypt_buffer(struct OpenCDMSession* session, GstBuffer* buffer, GstCaps* caps);
 
+/**
+ * \brief adds SVP related features to the caps structure (only if needed by the platform)
+ *
+ * \param caps GstCaps to be updated
+ *
+ * \return Zero on success, non-zero on error.
+ */
+
+    EXTERNAL OpenCDMError opencdm_gstreamer_transform_caps(GstCaps** caps);
```

Comment:
I guess this method does have an implementation on the current SVP adapter as well. Why was this not a nessecity in the previus setup? Is this not part of the implementation of the decrypt? What I am trying to say :-) Is this method really nessecary otr is this implementation leakage ? If the latetr, please remove, and move it into the decrypt call